watchtower-module-regex/src/main/java/com/tracelink/appsec/module/regex/service/RegexRuleService.java

```diff
@@ -26,6 +48,15 @@
 	 */
 	public RegexRuleService(@Autowired RegexRuleRepository ruleRepository) {
 		this.ruleRepository = ruleRepository;
+		PolymorphicTypeValidator v =
+				BasicPolymorphicTypeValidator.builder()
+						.allowIfBaseType(RuleDto.class)
+						.allowIfBaseType(Set.class)
+						.allowIfBaseType(RulesetDto.class)
+						.build();
+
+		jsonMapper =
```

Comment:
Weird formatting

Reply:
yeah... my ide is automatically doing that. I'll try to make it better